.eleventy.cjs

```diff
@@ -139,6 +134,21 @@ module.exports = function(eleventyConfig) {
             .filter(x => x && x !== '@patternfly/pfe-styles'),
         ],
       })));
+  /** Copy any component lightdom styles to the `assets` dir in the web root */
+  eleventyConfig.on('eleventy.before', async () =>
+    Promise.all(glob.sync('element/*/*-lightdom.css').map(x =>
+      fs.copyFile(x, path.join('_site', 'assets', path.basename(x))))));
```

Comment:
A couple of issues are resolved with this suggestion.  

- Corrects typo for base folder `elements/*` instead of `element/*`.
- The glob pattern needed `**` in the pattern `elements/**/*-lightdom.css` for subdirectories search
- `fs.copyFile` third parameter is a callback  unless you use `fs/promises` if you'd rather use the callback less version include `const fs = require('fs/promises');` and replace the callback version with `fs.copyFile(filePath, destPath);`

After the change is added `rh-footer-lightdom.css` is correctly moved to `_site/assets/lrh-footer-lightdom.css`


```Suggestion
  eleventyConfig.on('eleventy.before', async () =>
    Promise.all(glob.sync('elements/**/*-lightdom.css').map(filePath => {
      const destPath = path.join('_site', 'assets', path.basename(filePath));
      fs.copyFile(filePath, destPath, err => {
        if (err) {
          // eslint-disable-next-line no-console
          console.log('Error Copying lightdom css', err);
        }
      });
    })));
```

Reply:
you got it!